game-core/src/system/camera.rs

```diff
@@ -0,0 +1,48 @@
+use amethyst::renderer::Camera; 
+use amethyst::{
+    core::Transform,
+    ecs::{Join, ReadStorage, System, WriteStorage},
+};
+
```

Comment:
Try to avoid newlines between use statements. Otherwise rustfmt won't enforce consistent ordering.